modules/services/holo-auto-pilot.nix

```
@@ -0,0 +1,64 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.holo-auto-pilot;
+in
+
+{
+  options.services.holo-auto-pilot = {
+    enable = mkEnableOption "holo-auto-pilot";
+
+    install-list = mkOption {
+      type = types.attrs;
+    };
+
+    membrane-proofs = mkOption {
+      type = types.attrs;
+    };
+
+    package = mkOption {
+      default = pkgs.holo-auto-pilot;
+      type = types.package;
+    };
+
+    working-directory = mkOption {
+      default = "";
+    };
+  };
+
+  config = mkIf (cfg.enable) {
+    systemd.services.holo-auto-pilot = {
+      after = [ "network.target" "holochain.service" "hpos-holochain-api.service" "configure-holochain.service" ];
+      requisite = [ "holochain.service" "hpos-holochain-api.service" "configure-holochain.service" ];
```

```suggestion
      after = [ "network.target" "configure-holochain.service" ];
      requisite = [ "configure-holochain.service" ];
```
`configure-holochain` already depends on `holochain.service` and `hpos-holochain-api.service`